packages/blockchain-extension/resources/tutorials/developer-tutorials/privateData.md

```diff
@@ -104,71 +104,72 @@ It is also worth having a look at the collections.json file at the root of the f
 
 In this step, we will take you through the steps of creating a 2 org network. However, if you already have a 2 org network set up, feel free to use that instead.
 
-> Note: If you use your own network, you will need to change the MSP ID in the collections.json file once the contract has been generated.
+> Note: If you use your own network, add it as a collection in the collections.json file once the contract has been generated.
 
 1. Hover over the `FABRIC ENVIRONMENTS` panel on the left hand side of the screen and select the +.
 2. You will then be asked to `Select a method to add an environment`. Choose the `Create new from template` option.
 3. From the list of network configuration options, choose the `2 Org template (2 CAs, 2 peers, 1 channel)`.
-4. Give your network a name e.g. `newNetwork`. Whatever you find easy to identify! Then press `enter` which will trigger your network runtime to start.
+4. Give your network a name e.g. `newNetwork`. Whatever you find easy to identify!
```

Comment:
Can you re-add `Then press `enter` which will trigger your network runtime to start.`